Instruments/source/SyncTreeProducer.cxx

```diff
@@ -14,13 +14,16 @@ This file is part of https://github.com/hh-italian-group/h-tautau. */
 #include "hh-bbtautau/Analysis/include/SampleDescriptorConfigEntryReader.h"
 #include "hh-bbtautau/Analysis/include/SyncTupleHTT.h"
 #include "h-tautau/Analysis/include/SignalObjectSelector.h"
+//#include "h-tautau/Production/interface/TriggerTools.h"
 
 struct Arguments {
     REQ_ARG(std::string, mode);
     REQ_ARG(std::string, input_file);
     REQ_ARG(std::string, tree_name);
     REQ_ARG(std::string, period);
+    REQ_ARG(std::string, trigger_cfg);
     REQ_ARG(std::string, output_file);
+    REQ_ARG(std::vector<std::string>, trigger_patterns_vbf); //"HLT_VBF_DoubleLooseChargedIsoPFTau20_Trk1_eta2p1_Reg_v"
```

Comment:
definition of trigger patterns better to keep inside sync producer as it was before. Instead add boolean to indicate whatever to do AcceptAndMatch for vbf trigger (for example, for the moment we don't use them in the baseline, because we don't have the trigger SF).